### PR TITLE
[docs] - adds full dark mode to the docs

### PR DESCRIPTION
**Pull request by smolinari**
I hope this doesn't cause more work than necessary. 😄 

Dark mode works. I've left my boot file addition for SSR, however, I've commented out the `quasar.conf.js` entry and it still works (for me on Win10). 

Testing in Firefox, Chrome and Edge (all latest versions). It seems FF has an issue with carrying over cookies into new tabs or windows. Or, I have some secret setting set I don't know about. At any rate, dark looks good in all those browsers. 

Scott

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [x ] Feature
- [x ] Documentation
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [ ] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [ ] It's submitted to the `dev` branch and _not_ the `master` branch
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix: #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] It's been tested on a Cordova (iOS, Android) app
- [ ] It's been tested on a Electron app
- [ ] Any necessary documentation has been added or updated [in the docs](https://github.com/quasarframework/quasar/tree/dev/docs) (for faster update click on "Suggest an edit on GitHub" at bottom of page) or explained in the PR's description.

If adding a **new feature**, the PR's description includes:
- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

### CODE REVIEW

File: docs/quasar.conf.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ module.exports = function (ctx) {
     // --> boot files are part of "main.js"
     boot: [
       { path: 'gdpr', server: false },
+      // { path: 'darkMode', server: true },
```
Comment (lucasfernog) [inline docs/quasar.conf.js]: darkMode boot entry commented

Comment (smolinari) [reply docs/quasar.conf.js]: Yeah. Because it works without it, also in SSR mode. Strangely enough....

Scott

**Comment by ggedde** [comment]
Any Eta on this?
I have been looking around to see if I can find any good theme examples with dark mode, but I can't find any.  Having the docs have a switch for dark mode would be great and would fulfill that need.
Thanks

**Comment by smolinari** [comment]
@ggedde - Buggers. It seems I totally forgot I did this work and zapped (deleted) my repo. I'll rework the changes back into a new one and I'll create a new PR. This one needs to be closed. 

Once the new repo is available, you can clone it and run it in dev mode to see the dark mode in action. 

Scott

**Comment by smolinari** [comment]
New PR here: https://github.com/quasarframework/quasar/pull/6000

Scott

**Pull request closed without merging**